pkg/collector/corechecks/systemd/systemd.go

```diff
@@ -263,6 +271,17 @@ func (c *SystemdCheck) getSystemBusSocketConnection() (*dbus.Conn, error) {
 	return conn, err
 }
 
+func (c *SystemdCheck) submitVersion(conn *dbus.Conn) {
+	version, err := c.stats.GetVersion(conn)
+	if err != nil {
+		log.Debugf("Error collection version from the systemd: %v", err)
+		return
+	}
+	checkID := string(c.ID())
+	inventories.SetCheckMetadata(checkID, "version.raw", version)
+	log.Debugf("Submitted version %v for checkID %v", version, checkID)
```

Comment:
```suggestion
	log.Debugf("Submit version %v for checkID %v", version, checkID)
	inventories.SetCheckMetadata(checkID, "version.raw", version)
```

(nit) Having the debug before might be more helpful in case `SetCheckMetadata` fails.